package.json

```diff
@@ -7,7 +7,7 @@
     "npm": ">=7.24.2"
   },
   "dependencies": {
-    "@alephium/sdk": "0.0.5",
+    "@alephium/sdk": "^0.0.9",
```

Comment:
Didn't we say we wanted to fix the version and not use the `^`?

Reply:
We did indeed